docs/development.rst

```diff
@@ -96,6 +96,12 @@ Code style guide
 
 - First and foremost, the linters configured for the project must pass; this generally means following PEP-8 rules,
   as codified by: ``flake8``, ``black``, ``isort``, ``pyupgrade``.
+- The supported Python versions (and the code syntax to use) are listed in the ``tox.ini`` file in the ``tox/envlist``
```

Comment:
This is not true. The only authoritative source of truth for this is the setup.cfg python_requires entry.